chainerx_cc/chainerx/routines/manipulation_test.cc

```diff
@@ -839,7 +839,9 @@ TEST_THREAD_SAFE_P(ManipulationTest, SplitDefaultAxesIndices) {
     Array e1 = testing::BuildArray({1, 4}).WithData<int32_t>({0, 1, 2, 3});
     Array e2 = testing::BuildArray({1, 4}).WithData<int32_t>({4, 5, 6, 7});
 
-    Run([&]() { testing::CheckForward([](const std::vector<Array>& xs) { return Split(xs[0], {2}); }, {a}, {e1, e2}); });
+    Run([&]() {
+        testing::CheckForward([](const std::vector<Array>& xs) { return Split(xs[0], std::vector<int64_t>{1}); }, {a}, {e1, e2});
```

Comment:
Typo.
```suggestion
        testing::CheckForward([](const std::vector<Array>& xs) { return Split(xs[0], std::vector<int64_t>{2}); }, {a}, {e1, e2});
```

Reply:
This fix is intentional.
Please check the shapes.